local/bin/sh/build-vendor-js.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+echo "Bundling vendor JS dependencies..."
+
+ASSETS_VENDOR_DIR=assets/scripts/vendor/
+mkdir -p $ASSETS_VENDOR_DIR
+cp node_modules/jquery/dist/jquery.min.js $ASSETS_VENDOR_DIR
+cp node_modules/popper.js/dist/umd/popper.min.js $ASSETS_VENDOR_DIR
+cp node_modules/bootstrap/dist/js/bootstrap.min.js $ASSETS_VENDOR_DIR
+cp node_modules/bootstrap/dist/js/bootstrap.min.js.map $ASSETS_VENDOR_DIR
+
```

Comment:
do we need this script?

Reply:
Can't we just reference the files from node_modules instead? https://gohugo.io/hugo-pipes/js/#include-dependencies-in-packagejson--node_modules

Reply:
@davidejones i've tried the `hugo mod npm pack` as described there but wasn't having much luck (it is marked as experimental so perhaps there are more updates to come).  i will have another look at that however.

another thought i had was the `vendor.js` JS on corp/docs are bloated and a source of much unused code that is being downloaded.   the idea here was that we copy over these dependencies as needed.

Reply:
I feel like you should just be able to do `import mixitup from 'mixitup';` type syntax in the js file for node_modules libraries we are using and it get bundled in automatically. I could be wrong though or it could be a newer hugo version